assets/src/blocks/amp-story/edit.js

```diff
@@ -134,20 +135,48 @@ class EditPage extends Component {
 			autoAdvanceAfterHelp = __( 'Based on the duration of all animated blocks on the page', 'amp' );
 		}
 
+		const overlayStyle = {
+			width: '100%',
+			height: '100%',
+		};
+		if ( ! gradientBottomColor && overlayColor ) {
+			overlayStyle.backgroundColor = overlayColor;
+			overlayStyle.opacity = overlayOpacity / 100;
+		} else if ( gradientBottomColor ) {
+			const topColor = overlayColor ? overlayColor : 'transparent';
+			overlayStyle.backgroundImage = `linear-gradient(to bottom, ${ topColor }, ${ gradientBottomColor })`;
+			overlayStyle.opacity = overlayOpacity / 100;
+		}
+
 		return (
 			<Fragment>
 				<InspectorControls key="controls">
 					<PanelColorSettings
-						title={ __( 'Color Settings', 'amp' ) }
+						title={ __( 'Overlay Settings', 'amp' ) }
```

Comment:
```suggestion
						title={ __( 'Background Color', 'amp' ) }
```

If one just wants to use a solid background color without any gradient, I think this wording makes more sense.

Thanks to the opacity control further down, one will quickly discover that this can be used as an overlay for the background image.

Reply:
Hmm, I was thinking that `Overlay` is a word that is valid for both cases since Overlay doesn't necessarily have to be an overlay for an image -- it can be overlay for the Page block without the image, too. Also, technically it'll be a new Fill layer (aka overlay) and not use the background color.

Since the user will not necessarily open the Panel to see what's inside then maybe Background Color could be deceiving, it would be great if the title would be self-explanatory without having to know what's inside. Maybe there's another option instead?

Reply:
`Background Settings` maybe? Then it could be combined with the background media panel.

Technically it's a fill layer, but for the user it's basically a background color setting with some optional opacity.

For me, "background" is easier to grasp than "overlay", and also easier to localize.

Reply:
`Background Settings` makes sense 👍

Reply:
Since from the user perspective the net effect of this setting is providing a background color/fill, I would consider `Background Color` over `Settings. We want to make the language and UI as straightforward and intuitive as possible.